build.cmd

```diff
@@ -380,6 +390,45 @@ for /f "tokens=*" %%s in ('%DotNetCli% msbuild "%OptDataProjectFilePath%" /t:Dum
     set __IbcOptDataVersion=%%s
 )
 
+REM =========================================================================================
+REM ===
+REM === Generate source files for eventing
+REM ===
+REM =========================================================================================
+
+REM Find python and set it to the variable PYTHON
+echo import sys; sys.stdout.write(sys.executable) | (py -3 || py -2 || python3 || python2 || python) > %TEMP%\pythonlocation.txt 2> NUL
+set /p PYTHON=<%TEMP%\pythonlocation.txt
+if NOT DEFINED PYTHON (
+    echo %__MsgPrefix%Error: Could not find a python installation
+    exit /b 1
+)
+
+set __IntermediatesIncDir=%__IntermediatesDir%\src\inc
+set __IntermediatesEventingDir=%__IntermediatesDir%\eventing
+
+echo Laying out dynamically generated files consumed by the build system
```

Comment:
I appreciate you have these "echo" status messages. Can you please make sure they all start with `echo %__MsgPrefix%`, so we know where the message came from?

Reply:
Fixed in %__MsgPrefix%